### DISCUSSION TITLE
Component Data Serialized

### DISCUSSION THREAD

**Discussion by dustinlacewell** [Elderjs/elderjs#176] [Q&A] [answered]
I was just curious if component data returned from the route data function undergoes JSON de/serialization like it does when NextJS's `getStaticProps` are passed to the component.

**Comment by nickreese** [comment, marked as answer]
@dustinlacewell Great question. Data from the route's `data` function is passed as is. No serialization is done, so you can pass `function`s, `set`s, etc.

**Reply by dustinlacewell** [reply]
:open_mouth: Thanks!
